trio/_core/_run.py

```diff
@@ -1478,15 +1478,16 @@ class RunStatistics:
 # So this object can reference Runner, but Runner can't reference it. The only
 # references to it are the "in flight" callback chain on the host loop /
 # worker thread.
+
+
 @attr.s(eq=False, hash=False, slots=True)
```

Comment:
Hmm unintentional new whitespace?

Reply:
This is a part of one of the formatting rules that says that there should always be two newlines between any new definitions in the global scope